### PR TITLE
Updated Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
:-)

### CODE REVIEW

File: po/nl.po [hunk 1]
```diff
@@ -192,44 +191,39 @@ msgstr "Houd jezelf gezond."
...
 #: data/com.github.elfenware.badger.appdata.xml.in:37
 msgid "Add: 🎉️ Global switch to turn off all reminders 🎉️"
-msgstr ""
+msgstr "Toegevoegd: 🎉️ optie om alle herinneringen uit te schakelen 🎉️"
 
 #: data/com.github.elfenware.badger.appdata.xml.in:44
-#, fuzzy
 msgid "Add: Turkish translation (@libreajans)"
-msgstr "Toegevoegd: Deense vertaling (@siigdev)"
+msgstr "Toegevoegd: Turkse vertaling (@libreajeans)"
```
Comment (dar5hak) [inline po/nl.po]:
```suggestion
msgstr "Toegevoegd: Turkse vertaling (@libreajans)"
```

Comment (Vistaus) [reply po/nl.po]: You're right, I misread his name! Next time I'll use copy-and-paste. Sorry 'bout that, @libreajans ! And thank you for making me aware of it, @dar5hak :)

**Comment by dar5hak** [comment]
Whoa, that was quick! Thank you so much! :tada:

**Comment by Vistaus** [comment]
> Whoa, that was quick! Thank you so much!

You're welcome! :)

**Pull request merged**